docs/atola_taskforce.md

```diff
@@ -8,7 +8,7 @@ imager by [Atola Technology](https://atola.com/), designed for working
 with both good and damaged media with a focus on high-performance (up to
 18 parallel imaging sessions at 15 TB/hour).
 
-![Atola-taskforce.jpg](assets/images/500px-Atola-taskforce.jpg)
+<img src="assets/images/1000px-Atola-taskforce.jpg" title="Atola Taskforce"  alt="1000px-Atola-taskforce.jpg" align="right" />
```

Comment:
I think this caused the issue before - the path of "assets/images/..." in Markdown renders fine, but in `<img>` tags it doesn't work properly (the generated path is https://forensics.wiki/<page_name>/assets/images/whatever.jpg, when <page_name> shouldn't be included. An annoying thing is that it does render fine in preview, just not once the `gh-pages` part is generated and served by the live website. 

I think using Markdown syntax where possible should be our standard, and only use HTML tags where absolutely necessary (like if you needed to right-align a small image inside a block of text, like some pages do). In this example, it's a 1000px image by itself, so I think Markdown would work well.

Reply:
I was having some issues with alignment when using markdown.  Maybe that is local preview and not a markdown issue.  Ill have another look

Reply:
So I have issue previewing locally if I prepend `../` on to the path.  I took a look at the live site, specifically the [$mft.md](https://forensics.wiki/%24mft/) article.  The images are rendering there without prepending `../`.  I checked locally and the image render in the [$mft.md](https://forensics.wiki/%24mft/) file.  If I prepend `../` it breaks the images.

Reply:
Yes, I agree, when using the Markdown syntax (as in the $mft.md example), using `assets` works as expected. However, when you use `<img>` tags, the same path won't render (you need the `../assets` prefix) on the live site. It **won't** render in the preview, but it will on the live site, which is why it's confusing. I think it has to do with MkDocs using python-markdown vs whatever GitHub uses, but I'm not sure.  

See https://github.com/forensicswiki/wiki/blob/main/docs/lee_whitfield.md for an example of the live site rendering, but the preview not, when using a `<img>` tag.